src/upgrade/nc_upgrade_manager.js

```diff
@@ -279,7 +279,8 @@ class NCUpgradeManager {
             upgrade_package_version: this_upgrade.package_to_version,
             upgrade_history: {
                 ...upgrade_history,
-                successful_upgrades: [this_upgrade, ...successful_upgrades]
+                successful_upgrades: [this_upgrade, ...successful_upgrades],
+                last_failure: undefined
```

Comment:
Could you copy your comment here as well? ("// last_failure should be removed after a successful upgrade")